From: Haiyang Zhang <haiyangz@microsoft.com>
Date: Fri, 21 Apr 2023 10:06:57 -0700
Subject: net: mana: Rename mana_refill_rxoob and remove some empty lines
Patch-mainline: v6.4-rc1
Git-commit: 5c74064f43c291d9add2b436a2d70205b71a7cc7
References: bsc#1210551

Rename mana_refill_rxoob for naming consistency.
And remove some empty lines between function call and error
checking.

Signed-off-by: Haiyang Zhang <haiyangz@microsoft.com>
Signed-off-by: Jakub Kicinski <kuba@kernel.org>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/microsoft/mana/mana_en.c |    9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

--- a/drivers/net/ethernet/microsoft/mana/mana_en.c
+++ b/drivers/net/ethernet/microsoft/mana/mana_en.c
@@ -522,7 +522,6 @@ static int mana_pre_alloc_rxbufs(struct
 
 		da = dma_map_single(dev, va + mpc->rxbpre_headroom,
 				    mpc->rxbpre_datasize, DMA_FROM_DEVICE);
-
 		if (dma_mapping_error(dev, da)) {
 			put_page(virt_to_head_page(va));
 			goto error;
@@ -1468,7 +1467,6 @@ static void *mana_get_rxfrag(struct mana
 
 	*da = dma_map_single(dev, va + rxq->headroom, rxq->datasize,
 			     DMA_FROM_DEVICE);
-
 	if (dma_mapping_error(dev, *da)) {
 		put_page(virt_to_head_page(va));
 		return NULL;
@@ -1478,14 +1476,13 @@ static void *mana_get_rxfrag(struct mana
 }
 
 /* Allocate frag for rx buffer, and save the old buf */
-static void mana_refill_rxoob(struct device *dev, struct mana_rxq *rxq,
-			      struct mana_recv_buf_oob *rxoob, void **old_buf)
+static void mana_refill_rx_oob(struct device *dev, struct mana_rxq *rxq,
+			       struct mana_recv_buf_oob *rxoob, void **old_buf)
 {
 	dma_addr_t da;
 	void *va;
 
 	va = mana_get_rxfrag(rxq, dev, &da, true);
-
 	if (!va)
 		return;
 
@@ -1545,7 +1542,7 @@ static void mana_process_rx_cqe(struct m
 	rxbuf_oob = &rxq->rx_oobs[curr];
 	WARN_ON_ONCE(rxbuf_oob->wqe_inf.wqe_size_in_bu != 1);
 
-	mana_refill_rxoob(dev, rxq, rxbuf_oob, &old_buf);
+	mana_refill_rx_oob(dev, rxq, rxbuf_oob, &old_buf);
 
 	/* Unsuccessful refill will have old_buf == NULL.
 	 * In this case, mana_rx_skb() will drop the packet.
